cmd/qliksense/context.go

```diff
@@ -47,24 +47,27 @@ func setConfigsCmd(q *qliksense.Qliksense) *cobra.Command {
 		Short:   "set configurations into the qliksense context",
 		Example: `qliksense config set-configs <key>=<value>`,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return qliksense.SetConfigs(q, args)
+			return q.SetConfigs(args)
 		},
 	}
 	return cmd
 }
 
 func setSecretsCmd(q *qliksense.Qliksense) *cobra.Command {
 	var (
-		cmd *cobra.Command
+		cmd    *cobra.Command
+		secret bool
 	)
 
 	cmd = &cobra.Command{
 		Use:     "set-secrets",
 		Short:   "set secrets configurations into the qliksense context",
-		Example: `qliksense config set-secrets <key>=<value> --secret`,
+		Example: `qliksense config set-secrets <key>=<value> --secret=true`,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return qliksense.SetSecrets(q, args)
+			return q.SetSecrets(args, secret)
 		},
 	}
+	f := cmd.Flags()
+	f.BoolVar(&secret, "secret", false, "Whether secrets should be encrypted")
```

Comment:
secret always be encrypted !!

Reply:
Right! I forgot to fix the words here to be more explicit. Thanks for pointing out.